### PR TITLE
add Hyperloglog DT support and few minor test fixins

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Still Open Q's
- Figure out correct riak_pb tag
- Figure how best to support these changes and handle other clients!
- **expect buildbot** tests to fail b/c live_node tests will need the new HLL shit runnin'!

### CODE REVIEW

File: src/riakc_counter.erl [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 -ifdef(EQC).
 -include_lib("eqc/include/eqc.hrl").
--compile(export_all).
+-export([gen_type/0, gen_op/0]).
```
Comment (russelldb) [inline src/riakc_counter.erl]: heh, sometimes you like export_all, sometimes you don't? No need to change this back, I'm just making a comment.

Comment (zeeshanlakhani) [inline src/riakc_counter.erl]: i actually don't... I think the other one was just temp and will be changed :).

**Comment by russelldb** [comment]
Tests pass, works for me

+1

**Comment by nickelization** [comment]
We probably really want this in develop, but since master is behind develop I'll merge this here as-is, then do a separate PR to forward merge master.

**Pull request merged**